pkg/plugin/framework/server_mux.go

```diff
@@ -43,9 +45,20 @@ func newServerMux(logger logrus.FieldLogger) *serverMux {
 	}
 }
 
-// register registers the initializer for name.
+// register registers the initializer for the given name. Proper format
+// for the name is <namespace>/<name> and there can be no duplicates.
 func (m *serverMux) register(name string, f HandlerInitializer) {
-	// TODO(ncdc): return an error on duplicate registrations for the same name.
+	name = strings.ToLower(name)
+	if !strings.Contains(name, "/") {
```

Comment:
While this works, `/` is a valid name here. :)

Reply:
Argh true. Thanks.